sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Certificate.cs

```diff
@@ -1,60 +1,98 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using System.Text.Json;
 
 namespace Azure.Security.KeyVault.Certificates
 {
     /// <summary>
-    /// An Azure Key Vault certificate
+    /// An Azure Key Vault certificate.
     /// </summary>
-    public class Certificate : CertificateBase
+    public class Certificate : IJsonDeserializable
     {
+        private const string KeyIdPropertyName = "kid";
+        private const string SecretIdPropertyName = "sid";
+        private const string ContentTypePropertyName = "contentType";
+        private const string CERPropertyName = "cer";
+
+        /// <summary>
+        /// The Id of the certificate.
+        /// </summary>
+        public Uri Id => Properties.Id;
```

Comment:
Do we really need to add all these helper properties? Would it be too bad if we asked users to access them on Properties object?

Reply:
The code was originally that way, but we felt - across languages - that reflecting the identity properties on the outer object made sense since these identifying properties seem more than just "Properties" like expiration dates, etc. E.g. the `Certificate` has an identity.